server/routes/overview/overviewController.test.ts

```diff
@@ -25,6 +26,7 @@ describe('overviewController', () => {
   }
```

Comment:
Can we revert the changes to `overviewController.test.ts` and `overviewController.ts` please - I don't think we're quite ready for the controller to call the service yet as we've not fully planned/designed where the filtering/mapping/grouping etc will happen. This really is quite a complex piece of this overall feature, so we need to step back from it and make sure we have designed and understood it.

Reply:
(chances are the test and implementation will look like you've done them, but equally they might not, so safest at this stage to take it in small chunks and do each part iteratively. We can always get your test etc from git history as and when we get there, but I dont think this story should be doing it yet)